package.json

```
@@ -15,9 +15,13 @@
   "license": "MIT",
   "scripts": {
     "prepublish": "yarn run build",
-    "fix": "yarn prettier . --write",
-    "lint": "yarn prettier . --check && tslint -p ./tsconfig.json",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "lint": "run-p lint:*",
+    "lint:eslint": "eslint --ext=.ts .",
```

```suggestion
    "lint:eslint": "eslint --ext=.ts,.js,.cjs .",
```

Currently we have `.js` and `.cjs` too, let's add them all.